Context

Robin1711 opened the pull request:
Pull requirements for the features:
Anonymous login
Context full screen
Showing code in digit form and qrcode

There still is a bug in loading the words for an anonymous account.

yarchiT committed:
working context on top of master branch

Commit:
Changes on top of updated master

Commit:
Config.xml updated

Robin1711 committed:
manually merged codeShow

Robin1711 committed:
pushing branch code

Robin1711 committed:
Merge pull request #8 from mircealungu/show

Show

Robin1711 committed:
Merge pull request #9 from mircealungu/login

Login

mircealungu commented on .sign/signature1.xml:
```diff
@@ -8,7 +8,7 @@
 </Reference>
```
add to .gitignore .sign and .settings -- and any other file that is generated by the IDE

mircealungu commented on .tproject:
```diff
@@ -1,11 +1,12 @@
+        <blacklist/>
+        <resFallback autoGen="true"/>
+    </package>
+</tproject>
```
.gitignore .wgt

mircealungu commented on config.xml:
```diff
@@ -1,19 +1,19 @@
+    <feature name="http://tizen.org/feature/location.gps"/>
+
+<!-- Cf. https://developer.tizen.org/development/tutorials/web-application/w3chtml5supplementary-features/watch-application#project 
+	the tizen:category has to be set to this for the thing to be a watch face. -->
```
add comment to Readme.md~!!

mircealungu commented on css/style.css:
```diff
@@ -17,6 +17,154 @@ html,body {
+	background-size: 360px 360px;
+}
+.flagButton{
+	 border: 1px solid #e7e7e7;
```
indent~!

mircealungu commented on css/style.css:
```diff
@@ -17,6 +17,154 @@ html,body {
+	 border: 1px solid #e7e7e7;
+	border-radius: 6px;
+	box-shadow: 0 12px 16px 0 rgba(128,128,128,0.19), 0 12px 16px 0 rgba(128,128,128,0.19);
+	
```
no need for empty line!~

mircealungu commented on css/style.css:
```diff
@@ -17,6 +17,154 @@ html,body {
+}
+
+.langScreensButtons{
+width: 80px;
```
please look at formatting.

mircealungu commented on index.html:
```diff
@@ -6,45 +6,51 @@
     <meta name="description" content="Tizen Wearable Web Basic Template" />
 
-    <title>Time to Learn</title>
+    <title>Tizen Wearable Web Basic Application</title>
```
!!

mircealungu commented on index.html:
```diff
@@ -6,45 +6,51 @@
     <link rel="stylesheet" type="text/css" href="css/style.css" />
-    <script data-main="js/build/main-built.js" src="js/lib/require.js"></script> 
-    <!-- <script data-main="js/app/main.js" src="js/lib/require.js"></script> -->
+   <!-- <script data-main="js/build/main-built.js" src="js/lib/require.js"></script> -->
```
please add note to the documentation!

mircealungu commented on index.html:
```diff
@@ -6,45 +6,51 @@
-		<div id="nextButton">
-			<input type="submit" id="setNextButton" style="position:absolute; top:360; left:0px;" value=" ">
+		<div class="pageIndication">
+			<canvas id="pageIndicationQRCodeCanvas" width="360" height="20"></canvas>
```
refactor the with/height to css

mircealungu commented on index.html:
```diff
@@ -74,9 +80,10 @@
-				<input type="submit" id="profileButton" class="sButton" style="position:absolute; top:140px; left:80px;" value="profile">
-				<input type="submit" id="logOutButton" class="sButton" style="position:absolute; top:220px; left:80px;" value="log out">
+				<input type="submit" id="reverseButton" class="sButton" style="position:absolute; top:25px; left:80px;" value="reverse">
+				<input type="submit" id="profileButton" class="sButton" style="position:absolute; top:95px; left:80px;" value="profile">
```
consider redesign to four quadrants.

mircealungu commented on js/app/codeShow.js:
```diff
@@ -0,0 +1,164 @@
+
+	var NUMBER_OF_CODE_NUMBERS_ON_PAGE = 8;
+
+	function placePageIcon(page) {
```
add function comment saying what it does. 
if you can explain what's a page icon in the comments you're great!

mircealungu replied:
specify that we have only two pages for code / qucode!~

mircealungu commented on js/app/codeShow.js:
```diff
@@ -0,0 +1,164 @@
+	function placePageIcon(page) {
+		if (page === 1) {
+			ctx = document.getElementById("pageIndicationDigitsCanvas").getContext("2d");
+			ctx.clearRect(0, 0, SCREEN_WIDTH, ICON_HEIGHT);		
```
extract function that takes colors are arguments!

mircealungu commented on js/app/codeShow.js:
```diff
@@ -0,0 +1,164 @@
+		}
+	}
+	
+	function printCodeNumber(position, number){
```
add function comment

mircealungu commented on js/app/codeShow.js:
```diff
@@ -0,0 +1,164 @@
+	
+	function activateScreen(page) {
+		document.getElementById("mainPage").style.display = "none";
+		if (page === 1) {
```
use a constant like PAGE_TEXTUAL_CODE instead of 1,.

Commit:
Changes made during meeting to fix some bugs

Robin1711 committed:
Create .gitignore

Robin1711 committed:
Update .gitignore

Robin1711 committed:
Update README.md

Robin1711 committed:
minor fixes

Robin1711 committed:
major refactoring css and html

Robin1711 committed:
refactoring html and css

yarchiT committed:
Add hint to code show, refactor context

mircealungu commented:
dear people,

what's the status of this pull request? 
did you fix the bugs you still had to fix? 
can we help with anything? 

m.

yarchiT committed:
Refactor settings page

Robin1711 committed:
Fixed message and added comments

Robin1711 committed:
remove console.logs

Robin1711 committed:
remove console.log

Robin1711 committed:
words bug fix

yarchiT committed:
Merge branch 'context' into refCodeShow

Robin1711 committed:
update confit.xml

Added <tizen:category name="com.samsung.wmanager.WATCH_CLOCK"/>

Robin1711 committed:
Update index.html

uncommented <script data-main="js/build/main-built.js" src="js/lib/require.js"></script>

Nickborchers committed:
Merge pull request #12 from mircealungu/refCodeShow

Refactoring

mircealungu committed:
wgt files should not be versioned. they're binary

mircealungu committed:
tizen:profile name should be wearable

mircealungu committed:
script for building

mircealungu committed:
newline before eof

mircealungu committed:
ingoring *.wgt

yarchiT committed:
Updated interface and context, fixed time bug

yarchiT committed:
Update config.xml

Robin1711 committed:
small optimization creation QRcode

Robin1711 committed:
minor logical change

Robin1711 committed:
loadingPage tries

Robin1711 committed:
added comments

yarchiT committed:
working on loading screen

Commit:
fixed getAvailable

yarchiT committed:
working loading page

Robin1711 committed:
Merge branch 'context' into loadingPage

Robin1711 committed:
Merge pull request #13 from mircealungu/loadingPage

Loading page

Robin1711 committed:
making ready for pull request

yarchiT committed:
final version
